docs/using-jda/interactions.md

```diff
@@ -1,29 +1,36 @@
 # Interactions
 
+!!! note ""
```

Comment:
```suggestion
!!! note
```
I personally wouldn't remove titles in admonitions, instead just letting it take the default title for that admonition style like we have everywhere else on the wiki. This is just a minor nitpick for the sake of consistency. If you want to keep it like this I'm totally fine with that!

Reply:
I'd rather keep it as-is, it's just a little notice at the top and I was debating if it should even be in an admonition at all